Purge kotlin stdlib out of the classpath to not pollute with IJ IDEA's provided one.
<+>UTF-8
===================================================================
diff --git a/kotlin/internal/jvm/jvm_deps.bzl b/kotlin/internal/jvm/jvm_deps.bzl
--- a/kotlin/internal/jvm/jvm_deps.bzl	(revision 55f8585c27233d06160bf4f8bcbb6af029c2f239)
+++ b/kotlin/internal/jvm/jvm_deps.bzl	(date 1752054924661)
@@ -29,7 +29,6 @@
         associates = associate_deps,
     )
     dep_infos = (
-        [toolchains.kt.jvm_stdlibs] +
         associates.dep_infos +
         deps_java_infos +
         [_java_info(d) for d in deps]
